drivers/xen/gntdev-dmabuf.c

```diff
@@ -88,6 +115,68 @@ struct gntdev_dmabuf_priv {
 
 static void dmabuf_exp_release(struct kref *kref);
 
+static struct gntdev_dmabuf_wait_obj *
+dmabuf_map_wait_obj_find(struct gntdev_dmabuf_priv *priv, int fd)
+{
+	struct gntdev_dmabuf_wait_obj *obj, *ret = ERR_PTR(-ENOENT);
+
+	mutex_lock(&priv->lock);
+	list_for_each_entry(obj, &priv->map_wait_list, next)
+		if (obj->fd == fd) {
+			pr_debug("Found gntdev_dmabuf in the wait list\n");
```

Comment:
Do you really need this debug messages in the mainline?

Reply:
I don't know why, but if you take a look across gntdev-dmabuf file you will find that pr_debug is widely used to print error messages. I've just followed style